operator/cmd/mesh/testdata/manifest-generate/output/pilot_merge_meshconfig.yaml

```diff
@@ -0,0 +1,1978 @@
+# Base component is disabled.
```

Comment:
@howardjohn has made a good point about these test cases being much larger than necessary. we can prune this output file down to just the configmap by using the -p (config-path) flag.

Reply:
I don't think `-p` is defined for the generate command, only the profile dump command?

Reply:
yes, that's true. in that case would you mind writing a quick helper function that filters the output manifest based on object kind & name? you could use ParseK8sObjectsFromYAMLManifest and just scan thought each Object metadata for a match. it would be very helpful for creating more compact golden files going forward.